webapp/src/components/ParcelDetailPage/ParcelDetail/ParcelDetail.js

```diff
@@ -64,100 +64,105 @@ export default class ParcelDetail extends React.PureComponent {
-                <Grid.Column width={4}>Decentraland</Grid.Column>
                 <Grid.Column width={4}>
-                  <div className="address-wrapper" title={parcel.auction_owner}>
-                    <AddressBlock
-                      address={parcel.auction_owner}
-                      scale={4}
-                      hasTooltip={false}
-                    />&nbsp;
-                    <span className="short-address">
-                      {shortenAddress(parcel.auction_owner)}
-                    </span>
-                  </div>
+                  {t('parcel_detail.history.to')}
                 </Grid.Column>
               </Grid.Row>
-            ) : null}
-          </Grid>
-        </Grid.Column>
-      </Grid.Row>
+              {parcelPublications.map(publication => (
+                <Grid.Row
+                  key={publication.tx_hash}
+                  className="transaction-history-entry"
+                >
+                  <Grid.Column width={2}>
+                    {t('parcel_detail.history.sold')}
+                  </Grid.Column>
+                  <Grid.Column width={3}>
+                    <Mana amount={publication.price} />
+                  </Grid.Column>
+                  <Grid.Column width={3}>
+                    <BlockDate block={publication.block_number} />
+                  </Grid.Column>
+                  <Grid.Column width={4}>
+                    <div className="address-wrapper">
+                      <AddressBlock
+                        address={publication.owner}
+                        scale={4}
+                        hasTooltip={false}
+                      />&nbsp;
+                      <span className="short-address" title={publication.owner}>
+                        {shortenAddress(publication.owner)}
+                      </span>
+                    </div>
+                  </Grid.Column>
+                  <Grid.Column width={4}>
+                    <div className="address-wrapper" title={publication.buyer}>
+                      <AddressBlock
+                        address={publication.buyer}
+                        scale={4}
+                        hasTooltip={false}
+                      />&nbsp;
+                      <span className="short-address">
+                        {shortenAddress(publication.buyer)}
+                      </span>
+                    </div>
+                  </Grid.Column>
+                </Grid.Row>
+              ))}
+              {parcel.auction_price && parcel.auction_owner ? (
+                <Grid.Row className="transaction-history-entry">
+                  <Grid.Column width={2}>
+                    {t('parcel_detail.history.auction')}
+                  </Grid.Column>
+                  <Grid.Column width={3}>
+                    <Mana amount={parcel.auction_price} />
+                  </Grid.Column>
+                  <Grid.Column width={3}>
+                    {distanceInWordsToNow('2018-01-31T00:00:00Z')}
```

Comment:
Can we give a name to this constant? (`2018-01-31T00:00:00Z`) 🙏

Reply:
Also, not sure if worth it, but maybe the `TransactionHistory` or `TransactionHistoryRow` can be extracted?